test_scripts/Polices/build_options/TC003_REQUEST_PTU_Trigger_App_Excluded_PT_HTTP.lua

```diff
@@ -62,31 +66,28 @@ function Test:TestStep_StartNewSession()
 end
 
 function Test:TestStep_PTU_AppID_SecondApp_NotListed_PT()
-  local is_test_passed = true
   local correlationId = self.mobileSession1:SendRPC("RegisterAppInterface", config.application2.registerAppInterfaceParams)
 
+  self.mobileSession1:ExpectResponse(correlationId, { success = true, resultCode = "SUCCESS"})
+  self.mobileSession1:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
+
   EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application2.registerAppInterfaceParams.appName } })
-  :Do(function()
-
-    EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate",
-          { status = "UPDATE_NEEDED" }, {status = "UPDATING"}):Times(2)
-    :Do(function(_,data)
-      if(data.params.status == "UPDATE_NEEDED") then
-        is_test_passed = testCasesForPolicyTableSnapshot:verify_PTS(true,
-                { config.application1.registerAppInterfaceParams.appID, config.application2.registerAppInterfaceParams.appID, },
-                {config.deviceMAC},
-                {""},
-                "print")
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", { status = "UPDATE_NEEDED" }, {status = "UPDATING"}):Times(2)
+
+  self.mobileSession1:ExpectNotification("OnSystemRequest")
```

Comment:
@istoimenova, Since 2 apps are registered, OnSystemRequest() can be sent to any of them.
There are 2 options at least are possible:
1) Redesign script for only 1 app (with usage of connecttest_resumption)
2) Add expectation for 2nd app